influx repl panics when running queries calling the `to()` function

Open a influx repl and run the following Flux script:
```
import "csv"
inData = "
#datatype,string,long,dateTime:RFC3339,string,string,string,string,string,double
#group,false,false,false,true,true,true,true,true,false
#default,_result,,,,,,,,
,result,table,_time,_measurement,_field,device,fstype,host,_value
,,0,2018-05-22T00:00:00Z,disk,percentage,disk1s1,apfs,host.local,67.1
"

csv.from(csv: inData) |> to(org: "myorg", bucket: "buc")
```

the REPL will panic:
```
panic: interface conversion: interface {} is nil, not influxdb.StorageDependencies

goroutine 1 [running]:
github.com/influxdata/influxdb/v2/query/stdlib/influxdata/influxdb.GetStorageDependencies(...)
	influxdb/query/stdlib/influxdata/influxdb/dependencies.go:29
github.com/influxdata/influxdb/v2/query/stdlib/influxdata/influxdb.createToTransformation(0xd55b0e2da64b7c9d, 0x7833c7182b12b4a1, 0x0, 0x64b1f20, 0xc0001d60c8, 0x64d43a0, 0xc000c58540, 0xc000ad1088, 0x400e8c6, 0xc000c46800, ...)
	influxdb/query/stdlib/influxdata/influxdb/to.go:241 +0x550
github.com/influxdata/flux/execute.(*createExecutionNodeVisitor).Visit(0xc000cb5b40, 0x64e8f60, 0xc0003f81b0, 0x64e8f60, 0xc0003f8240)
	/go/pkg/mod/github.com/influxdata/flux@v0.67.1-0.20200512221521-be8c67e907a2/execute/executor.go:210 +0x8db
github.com/influxdata/flux/plan.walk(0x64e8f60, 0xc0003f81b0, 0x62c5830, 0xc000ad1550, 0xc000ad13a8, 0xc000cc0600, 0x60)
	/go/pkg/mod/github.com/influxdata/flux@v0.67.1-0.20200512221521-be8c67e907a2/plan/walk.go:88 +0x19e
github.com/influxdata/flux/plan.(*Spec).BottomUpWalk(0xc000c582c0, 0xc000ad1550, 0xc000cc0600, 0x53275e0)
	/go/pkg/mod/github.com/influxdata/flux@v0.67.1-0.20200512221521-be8c67e907a2/plan/walk.go:60 +0x2fd
github.com/influxdata/flux/execute.(*executor).createExecutionState(0xc0001d60f0, 0x64cb1e0, 0xc000cd2ff0, 0xc000c582c0, 0xc000cfe4b0, 0x53275e0, 0xc0001d60f0, 0x64b4f60)
	/go/pkg/mod/github.com/influxdata/flux@v0.67.1-0.20200512221521-be8c67e907a2/execute/executor.go:104 +0x18c
github.com/influxdata/flux/execute.(*executor).Execute(0xc0001d60f0, 0x64cb1e0, 0xc000cd2ff0, 0xc000c582c0, 0xc000cfe4b0, 0x7, 0x0, 0x0, 0x0)
	/go/pkg/mod/github.com/influxdata/flux@v0.67.1-0.20200512221521-be8c67e907a2/execute/executor.go:70 +0x5d
github.com/influxdata/flux/lang.(*Program).Start(0xc000cd2f60, 0x64cb1e0, 0xc000cd2f90, 0xc000cfe4b0, 0xc000cd2f90, 0x6496f60, 0xc000cd2f60, 0x0)
	/go/pkg/mod/github.com/influxdata/flux@v0.67.1-0.20200512221521-be8c67e907a2/lang/compiler.go:298 +0x1f8
github.com/influxdata/flux/repl.(*REPL).doQuery(0xc000a99c80, 0x64cb120, 0xc000c58240, 0xc0003f8090, 0x64d4320, 0xc000a9a900, 0x0, 0x0)
	/go/pkg/mod/github.com/influxdata/flux@v0.67.1-0.20200512221521-be8c67e907a2/repl/repl.go:229 +0x205
github.com/influxdata/flux/repl.(*REPL).executeLine(0xc000a99c80, 0xc000c98000, 0x17e, 0x0, 0xc000ad19b0)
	/go/pkg/mod/github.com/influxdata/flux@v0.67.1-0.20200512221521-be8c67e907a2/repl/repl.go:188 +0x372
github.com/influxdata/flux/repl.(*REPL).input(0xc000a99c80, 0xc000c98000, 0x17e)
	/go/pkg/mod/github.com/influxdata/flux@v0.67.1-0.20200512221521-be8c67e907a2/repl/repl.go:139 +0x46
github.com/c-bata/go-prompt.(*Prompt).Run(0xc000a9cb80)
	/go/pkg/mod/github.com/c-bata/go-prompt@v0.2.2/prompt.go:81 +0x6b4
github.com/influxdata/flux/repl.(*REPL).Run(0xc000a99c80)
	/go/pkg/mod/github.com/influxdata/flux@v0.67.1-0.20200512221521-be8c67e907a2/repl/repl.go:80 +0x1d6
main.replF(0xc000a8f900, 0xc000a99500, 0x0, 0x6, 0xc000c2bd68, 0x4a28cc4)
	influxdb/cmd/influx/repl.go:60 +0x1c3
main.checkSetupRunEMiddleware.func1.1(0xc000a8f900, 0xc000a99500, 0x0, 0x6, 0x0, 0x0)
	influxdb/cmd/influx/main.go:344 +0x74
github.com/spf13/cobra.(*Command).execute(0xc000a8f900, 0xc000a99440, 0x6, 0x6, 0xc000a8f900, 0xc000a99440)
	/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:826 +0x460
github.com/spf13/cobra.(*Command).ExecuteC(0xc000157680, 0x0, 0x0, 0xc000157680)
	/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:914 +0x2fb
github.com/spf13/cobra.(*Command).Execute(...)
	/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:864
main.main()
	influxdb/cmd/influx/main.go:32 +0x45
exit status 2
```

--- a/query/stdlib/influxdata/influxdb/dependencies.go
+++ b/query/stdlib/influxdata/influxdb/dependencies.go
@@ -26,6 +26,9 @@ func (d StorageDependencies) Inject(ctx context.Context) context.Context {
 }
 
 func GetStorageDependencies(ctx context.Context) StorageDependencies {
+	if ctx.Value(dependenciesKey) == nil {
+		return StorageDependencies{}
+	}
 	return ctx.Value(dependenciesKey).(StorageDependencies)
 }
 

--- a/query/stdlib/influxdata/influxdb/to.go
+++ b/query/stdlib/influxdata/influxdb/to.go
@@ -241,8 +241,15 @@ func createToTransformation(id execute.DatasetID, mode execute.AccumulationMode,
 	}
 	cache := execute.NewTableBuilderCache(a.Allocator())
 	d := execute.NewDataset(id, mode, cache)
-	deps := GetStorageDependencies(a.Context()).ToDeps
-	t, err := NewToTransformation(a.Context(), d, cache, s, deps)
+	deps := GetStorageDependencies(a.Context())
+	if deps == (StorageDependencies{}) {
+		return nil, nil, &flux.Error{
+			Code: codes.Unimplemented,
+			Msg:  "cannot return storage dependencies; storage dependencies are unimplemented",
+		}
+	}
+	toDeps := deps.ToDeps
+	t, err := NewToTransformation(a.Context(), d, cache, s, toDeps)
 	if err != nil {
 		return nil, nil, err
 	}

